### PR TITLE
#Hacktoberfest

### PR DESCRIPTION
**Pull request by beckerhe**
Am I doing this right? Probably not. I guess I'm also way too late to get a t-shirt. (Ref: https://twitter.com/shitoberfest)

But the whole discussion reminded me that we should update the external docs once in a while. So this is now
updating `DEVELOPMENT.md`

Commit message:
```
Some things in DEVELOPMENT.md have been outdated for a while.

This commit should fix that.

Apart from minor fixes it's mainly trying to explain the confusing
state of Windows profiling and Linux remote profiling,
and the `--local` flag.
```

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -73,7 +121,8 @@ style.
 `bootstrap-orbit.{sh,ps1}` performs all the tasks which have to be done once per developer machine.
 This includes:
 * Installing system dependencies
-* Installing conan if necessary.
+* Installing the correct version of conan if necessary.
+* Installing the conan configuration (which changes rarely)
```
Comment (ronaldfw) [inline DEVELOPMENT.md:125]: Add "." at the end for consistency with previous bullet point?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -86,6 +135,11 @@ Whenever the dependencies change you have to call `build.{sh,ps1}` again.
 A dependency change might be introduced by a pull from upstream or by a switch
 to a different branch.
 
+It might occur to you that even though you called `build{sh,ps1}` your build still
```
Comment (ronaldfw) [inline DEVELOPMENT.md:138]: ".' is missing in build{sh,ps1}

**Review by ronaldfw — APPROVED** [review]
LGTM (modulo minor nits) and thanks so much for improving our docs!

I don't know if you can still get a #hacktoberfest t-shirt, but we should consider Orbit swag.

**Review by pierricgimmig — APPROVED** [review]
Hey Henning, thanks for taking the time to revamp our doc. This is definitely worth a t-shirt!

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -1,13 +1,29 @@
 # Getting started with development
 
+Orbit consists of two parts - the frontend and the collector, also called the service.
+The collector is responsible for instrumenting the target process and recording
+profiling events which are then streamed to the frontend, also called the UI.
+
+The communication between frontend and collector is handled by a [gRPC](https://grpc.io/)
+connection. gRPC uses HTTP 2.0 as its base communication layer. When talking to a Stadia
+Instance we wrap that once more into an SSH tunnel.
```
[the remark this answers is not in the document]
Comment (beckerhe) [reply DEVELOPMENT.md]: I agree. Thanks. Fixed.

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -1,13 +1,29 @@
...
+
 ## Platforms
 
-Windows 10 and Linux are supported.
+The frontend is supported on Windows 10 and Linux. The collector currently
+only works on Linux.
+
+Previous versions of Orbit supported profiling on Windows, but due to the
+priority shift towards Stadia this support is currently in a non-working state.
+There are plans to bring it back, but at that point we can't commit to any time
```
[the remark this answers is not in the document]
Comment (beckerhe) [reply DEVELOPMENT.md]: done.

File: DEVELOPMENT.md (hunk 2 above)
Comment (beckerhe) [reply DEVELOPMENT.md]: done.

File: DEVELOPMENT.md (hunk 1 above)
Comment (beckerhe) [reply DEVELOPMENT.md]: done.

**Comment by beckerhe** [comment]
Thank you both for the review. I'm looking forward to Orbit swag. :sweat_smile:

**Pull request merged**